spec/unit/provider/oneview_san_manager_synergy_spec.rb

```diff
@@ -124,9 +124,16 @@
         allow(resourcetype).to receive(:get_all).with(anything).and_return([test])
         provider.exists?
       end
-      it 'should be able to run through self.instances' do
+
+      it 'should be able to run through instances' do
```

Comment:
This test specifically seems to be useless, it runs through instances in the test bellow and provider is whats being tested. Could remove this it entirely.